lfx-mentorship/2021/03-Fall/project_ideas.md

```diff
@@ -25,3 +25,20 @@ Project maintainers and mentors, please submit the ideas below (under the Propos
 - Mentor(s): [@Krasi Georgiev](https://github.com/krasi-georgiev)
 - Issue: <https://github.com/prometheus/prometheus/issues/3416>
 
+### Proposed Project ideas
+
+#### Vitess
+
+##### Add complete parsing support for MySQL constructs
+
+- Description: Vitess is a database clustering system for horizontal scaling of MySQL. One of the key goals of Vitess is to emulate MySQL behavior even while running multiple MySQL instances so that ORMs and frameworks work seamlessly. Vitess has its own in-built SQL-parser which it uses to understand the query and represent as structs for further processing. As of now, a lot of MySQL structs are not parsed and result in syntax errors. For example, we do not have complete support to parse [partition constructs](https://dev.mysql.com/doc/refman/5.7/en/partitioning-overview.html). The task of the mentee would be add parsing support for such constructs. 
```

Comment:
```suggestion
- Description: Vitess is a database clustering system for horizontal scaling of MySQL. One of the key goals of Vitess is to emulate MySQL behavior even while running multiple MySQL instances so that ORMs and frameworks work seamlessly. Vitess has its own in-built SQL-parser which it uses to understand the query and represent as structs for further processing. As of now, a lot of MySQL constructs are not parsed and result in syntax errors. For example, we do not have complete support to parse [partition constructs](https://dev.mysql.com/doc/refman/5.7/en/partitioning-overview.html). The task of the mentee would be to add parsing support for such constructs. 
```